### ISSUE TITLE
[Request] RAdamScheduleFree optimizer support

### ISSUE THREAD

**Issue by guy907223982** [67372a/LoRA_Easy_Training_Scripts#78]
Not sure if it was just renamed as there are many but the original Easy Training Scrips has RAdamScheduleFree. This one doesn't. I found it works the best with style loras so I would like to have it added. Thanks.

**Comment by 67372a** [comment]
@guy907223982 implemented. I changed silent_sgd_phase=False by default instead of True, if it's true, step two loss NaNs, this is true of baseline RAdamScheduleFree in the training scripts as well, no idea why, might be an upstream defect, but just keep in mind.

**Comment by 67372a** [comment]
Fixed the bug causing NaN with silent_sgd_phase=True, as such I set it to True by default again.
